src/loaders/google_map_loader.js

```diff
@@ -14,66 +10,51 @@ const _customPromise = new Promise((resolve) => {
 
 // TODO add libraries language and other map options
 export default (bootstrapURLKeys, heatmapLibrary) => {
-  if (!$script_) {
-    $script_ = require('scriptjs'); // eslint-disable-line
-  }
-
   // call from outside google-map-react
-  // will be as soon as loadPromise_ resolved
+  // will be as soon as loadPromise resolved
   if (!bootstrapURLKeys) {
     return _customPromise;
   }
 
-  if (loadPromise_) {
-    return loadPromise_;
+  if (!bootstrapURLKeys.libraries) {
+    bootstrapURLKeys.libraries = [];
   }
```

Comment:
@jpoehnelt Hey! Just wondering, why did we remove this line? I think this its causing this issue https://github.com/google-map-react/google-map-react/issues/954